specification/carbon/Carbon.Management/tspconfig.yaml

```diff
@@ -0,0 +1,48 @@
+parameters:
+  "service-dir":
+    default: "sdk/carbonoptimization"
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    emitter-output-dir: "{project-root}/.."
+    azure-resource-provider-folder: "resource-manager"
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/main.json"
+  "@azure-tools/typespec-python":
+    package-dir: "azure-mgmt-carbonoptimization"
+    package-name: "{package-dir}"
+    namespace: "azure.mgmt.carbonoptimization"
+    flavor: azure
+    generate-test: true
+    generate-sample: true
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.ResourceManager.CarbonOptimization"
+    clear-output-folder: true
+    model-namespace: false
+    namespace: "{package-dir}"
+    flavor: azure
+  "@azure-tools/typespec-ts":
+    is-modular-library: true
+    generate-metadata: true
+    experimental-extensible-enums: true
+    package-dir: "arm-carbonoptimization"
+    flavor: "azure"
+    package-details:
+      name: "@azure/arm-carbonoptimization"
+  "@azure-tools/typespec-java":
+    package-dir: "azure-mgmt-carbonoptimization"
+    namespace: "com.microsoft.azure.management.carbonoptimization"
```

Comment:
package and namespace is wrong

please follow the guide, and sample here https://github.com/Azure/azure-rest-api-specs/blob/main/specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml